DataFormats/Reconstruction/include/ReconstructionDataFormats/PID.h

```diff
@@ -99,12 +100,14 @@ class PID
   static constexpr ID Photon = 10;
   static constexpr ID K0 = 11;
   static constexpr ID Lambda = 12;
-  static constexpr ID HyperTriton = 13;
   static constexpr ID Hyperhydrog4 = 14;
   static constexpr ID XiMinus = 15;
   static constexpr ID OmegaMinus = 16;
+  // static constexpr ID FirstExt = PI0;
+  static constexpr ID Hyperhelium4 = 17;
+  static constexpr ID HyperTriton = 18;
   static constexpr ID FirstExt = PI0;
-  static constexpr ID LastExt = OmegaMinus;
+  static constexpr ID LastExt = Hyperhelium4;
```

Comment:
Why are you changing the IDs of existing particles? This is not backward-compatible. Also, with the `NIDsTot = 18` the specific IDs should span 0-17, while you reassign HyperTriton from 13 to 18.